bin/templating.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+# --- Setup ------------------------------------------------------------------ #
+
+set -eo pipefail
+
+bin_dir=$(dirname "${BASH_SOURCE[0]}")
+value=${TEMPLATE}
+
+# --- Import ----------------------------------------------------------------- #
+
+# shellcheck source=./lib/actions.sh
+source "${bin_dir}/../lib/actions.sh"
+
+# --- Script ----------------------------------------------------------------- #
+
+group_start "Templating..."
+
+debug "input '${value}'"
```

Comment:
It's a bit odd that this log group has no contents (without debug logging enabled). Maybe this particular log statement can be converted into:

```suggestion
info "input '${value}'"
```

Reply:
Resolved in 190211a052c9361616595ae37bd3019fd1a72f67 by removing the log group instead.